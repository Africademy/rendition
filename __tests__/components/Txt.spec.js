/* globals expect, test */
import React from 'react'
import renderer from 'react-test-renderer'
import {
  Provider,
  Txt
} from '../../dist'

test('Txt renders correctly', () => {
  const component = renderer.create(
    <Provider>
      <Txt>Hello world</Txt>
    </Provider>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
