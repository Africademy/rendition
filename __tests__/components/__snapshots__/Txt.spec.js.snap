// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Txt renders correctly 1`] = `
<div
  className="sc-bdVaJa iLLxmY"
>
  <div
    className="sc-htoDjs jslKMR sc-gzVnrw jCPitn"
  >
    Hello world
  </div>
</div>
`;
